readthedocs/audit/tasks.py

```diff
@@ -0,0 +1,31 @@
+"""Celery tasks related to audit logs."""
+
+import logging
+
+from django.conf import settings
+from django.utils import timezone
+
+from readthedocs.audit.models import AuditLog
+from readthedocs.worker import app
+
+log = logging.getLogger(__name__)
+
+
+@app.task(queue='web')
+def delete_old_personal_audit_logs(days=None):
+    """
+    Delete personal security logs older than `days`.
+
+    If `days` isn't given, default to ``RTD_AUDITLOGS_DEFAULT_RETENTION_DAYS``.
```

Comment:
Yea, I'd be 👍  on using the organization plan here, so users get the benefit of the organization's longer subscription time. I think this makes the most sense.

Reply:
I have overridden this task on .com, since there is code there that I'm using, but I'll move that code to .org soon. This PR still needs to be merged, it does the deletion for logs from .org where we don't have organizations, the other PR on .com overrides this.

Reply:
Cool! I think we should have all the code in just one project (here) and check for `RTD_ALLOW_ORGANIZATION` to decide what's the logic to run on each case.